README.md

```diff
@@ -18,667 +17,216 @@ replacement for common `pip` and `pip-tools` workflows.
   </picture>
 </p>
 
-<p align="center">
-  <i>Installing the Trio dependencies with a warm cache.</i>
-</p>
```

Comment:
Why remove?

Reply:
Not intentional, should be replaced with new benchmarks.